vignettes/contribution_model.Rmd

```diff
@@ -0,0 +1,58 @@
+---
+title: "Contribution to admiral"
+output: 
+  rmarkdown::html_vignette
+    
+vignette: >
+  %\VignetteIndexEntry{contribution_model}
+  %\VignetteEngine{knitr::rmarkdown}
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+# Goal: Enable users of admiral to contribute and test a contribution model for admiral.
```

Comment:
This is not really a title. How about we start with `# Introduction` as in all other vignettes and have this as the first sentence?